src/ebonite/client/base.py

```diff
@@ -107,9 +107,22 @@ def get_model(self, model_name: str, task: TaskVar, project: ProjectVar = None,
             model.load()
         return model
 
-    @abstractmethod
-    def build_service(self, name: str, model: 'core.Model', **kwargs):
-        pass
+    def build_service(self, name: str, model: 'core.Model', **kwargs) -> 'core.Image':
+        """
+        Builds image of model service and stores it to repository
+
+        :param name: name of image to build
+        :param model: model to wrap into service
+        :return: :class:`~ebonite.core.objects.Image` instance representing built image
+        """
+        if 'server' not in kwargs:  # by default we use uwsgi flask server
+            from ebonite.ext.flask.helpers import build_model_flask_docker
+            kwargs = {k: v for k, v in kwargs.items() if k in {'force_overwrite', 'debug'}}
+            image = build_model_flask_docker(name, model, **kwargs)
```

Comment:
Maybe we need a way to choose which method to use for building? Or maybe it should be a part of Server abstraction - how to build it?

Reply:
Building and running is somewhat coupled. But current `Server` abstraction is focused on running only and its implementations are already a bit bloated. I don't think it makes sense to put more responsibilites here.

As for `build_*` methods themselves the problem is that they have incompatible interface. Flask wrapper hardcodes some properties for `build_model_docker` and discards some other settings of this helper (I don't know whether this was by accident or for purpose).

Reply:
i bet it was an accident :) 
there is a simple user story for this case - when we building and deploying services from UI, we need them to be production-ready, and if tomorrow we will have working UI, it could not be done. so I suggest we postpone this (as always) and create a task in jira (i'm counting on you)

Reply:
In fact we somewhat could: if we do not pass server then "production" Flask is used, if we pass aiohttp then it is used (aiohttp has no separate debug/production versions, I treat existing version as "production"). But I agree with you that we should somehow make things more transparent in this direction. Reported EBNT-253 for this.